src/pages/myCollectionPage/MyCollectionPage.tsx

```diff
@@ -9,8 +9,8 @@ const MyCollectionPage = () => {
       <h3>moja kolekcja</h3>
       <CallToActionAsLink
         linkTo={"/dodaj-roślinkę"}
-        color={Color.cream}
-        backgroundColor={Color.mediumGreen}
+        color={theme.colors.cream}
+        backgroundColor={theme.colors.mediumGreen}
```

Comment:
move those two to styled component please

Reply:
fixed